jenkins/databricks/build.sh

```diff
@@ -297,21 +310,16 @@ run_mvn_cmd()
             -Dversion=$SPARK_VERSION_TO_INSTALL_DATABRICKS_JARS \
             -Dpackaging=jar
     done
-
-    # Build the RAPIDS plugin by running package command for databricks
-    mvn -B -Ddatabricks -Dbuildver=$BUILDVER clean package -DskipTests $MVN_OPT
 }
 
 ##########################
-# Main Script starts here
+# Main script starts here
 ##########################
 
 initialize
-set_sw_versions
-set_jars_prefixes
-set_dep_jars
-run_mvn_cmd
+install_dependencies
```

Comment:
can you add a variable to allow skipping install_dependencies?  that way DEV can say something like SKIP_DEP_INSTALL=1 ./jenkins/databricks/build.sh.  default value should to be not skip

Reply:
that's a good idea. As a workaround, my current flow boils down to running build.sh only once and then just work Maven directly:
```
mvn package -pl tests,integration_tests -am -DskipTests -Ddatabricks -Dbuildver=3XYdb
```

Reply:
Yeah , sounds useful addition.
 I made the changes but I will test it on my cluster before pushing the latest.